src/machine/usb/config.go

```diff
@@ -0,0 +1,14 @@
+package usb
+
+type EndpointConfig struct {
+	No        uint8
```

Comment:
This might be better named `Num` or perhaps `ID`?

Reply:
Perhaps `Index` would be a better name.

Reply:
Renamed.